src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -0,0 +1,45 @@
+class Workflow::Step::ConfigureRepositories < Workflow::Step
+  validate :validate_repositories
+
+  def call(_options = {})
+    return unless valid?
+
+    repositories.each do |repository|
+      target_repository = find_or_create_repository(repository, target_branch_project)
+      target_repository.architectures.destroy_all
```

Comment:
careful. 

```
[4] pry(main)> repo = Project.find_by_name('home:hennevogel:playground').repositories.first
=> #<Repository:0x0000561407490c20
 id: 3330457,
 db_project_id: 781679,
 name: "openSUSE_Leap_15.3",
 remote_project_name: "",
 rebuild: nil,
 block: nil,
 linkedbuild: nil,
 required_checks: []>
[5] pry(main)> repo.architectures.first.class
=> Architecture(id: integer, name: string, available: boolean)
```

Reply:
Oh right :facepalm:, it's `RepositoryArchitecture`!